mypy/stubgen.py

```diff
@@ -172,6 +172,11 @@ def find_module_path_and_all(module: str, pyversion: Tuple[int, int],
             raise SystemExit(
                 "Can't find module '{}' (consider using --search-path)".format(module))
         module_all = None
+    if not (
+        module_all is None or
+        (isinstance(module_all, list) and all(isinstance(v, str) for v in module_all))
+    ):
+        raise CantImport(module)
```

Comment:
This error might be confusing, because we were able to import the module, but it contains a critical error. Maybe add a message (explaining cause) to this exception class?